src/Framework/Microsoft.Build.Framework.csproj

```diff
@@ -15,7 +15,7 @@
       an imported package. This suppression should be removed if/when the project is migrated to enable nullable
       reference types.
     -->
-    <NoWarn>$(NoWarn),CS8632</NoWarn>
+    <NoWarn>$(NoWarn);CS8632</NoWarn>
```

Comment:
Sorry to merge conflict but since you pointed this out: #7426.

Reply:
I undid this change so prevent a merge conflict.